ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -138,7 +143,39 @@ object LedgerApiTestTool {
             commonConfig.withDarFile(resourceAsFile(semanticTestsResource))
       }
     )
-    Map(semanticTestsRunner)
+    val packageManagementServiceIT = lazyInit(
```

Comment:
I'd rather we don't add tests to the "default set of tests" until we have the tests also running against sandbox via the test tool. This is even more the case for tests that we semi-instantiate but don't make available anyway, like the `PartyManagementServiceIT` test.

Reply:
Moved to optional part.